backend/api/src/instrument.ts

```
@@ -1,3 +1,4 @@
+/// <reference path="./types/express.d.ts" />
```

ESLint

[error] Do not use a triple slash reference for ./types/express.d.ts, use `import` style instead.

(@typescript-eslint/triple-slash-reference)